### ISSUE TITLE
Test suit fails because py module is missing

### ISSUE THREAD

**Issue by mj0nez** [bo4e/BO4E-python#467]
**After cloning the repository and running ``tox`` the test suit fails because the ``py`` module seems to be missing.**

## Steps To Reproduce
```
git clone ...
tox -e tests
```
or
```
tox -r -e tests
```

### What you expected

```
============================================================================================ test session starts ============================================================================================
platform win32 -- Python 3.10.10, pytest-7.3.1, pluggy-1.0.0
cachedir: .tox\tests\.pytest_cache
rootdir: C:\Users\marce\repositories\BO4E-python
configfile: pyproject.toml
testpaths: tests
plugins: datafiles-3.0.0
collected 291 items

...
tests\test_zeitreihe.py ..                                                                                                                                                                             [ 97%] 
tests\test_zeitreihenwert.py ....                                                                                                                                                                      [ 98%] 
tests\test_zeitreihenwertkompakt.py ....                                                                                                                                                               [100%]

============================================================================================ 291 passed in 1.25s ============================================================================================ 
tests: OK ✔ in 29.42 seconds

```

### What actually happened
```
============================================================================================ test session starts ============================================================================================
platform win32 -- Python 3.10.10, pytest-7.3.1, pluggy-1.0.0
cachedir: .tox\coverage\.pytest_cache
rootdir: C:\Users\marce\repositories\BO4E-python
configfile: pyproject.toml
testpaths: tests
plugins: datafiles-3.0.0
collected 263 items / 6 errors

================================================================================================== ERRORS =================================================================================================== 
__________________________________________________________________________________ ERROR collecting tests/test_adresse.py ___________________________________________________________________________________ 
ImportError while importing test module 'C:\Users\marce\repositories\BO4E-python\tests\test_adresse.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\..\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests\test_adresse.py:4: in <module>
    from py._path.local import LocalPath  # type:ignore[import]
E   ModuleNotFoundError: No module named 'py._path'; 'py' is not a package
__________________________________________________________________________________ ERROR collecting tests/test_angebot.py ___________________________________________________________________________________ 
ImportError while importing test module 'C:\Users\marce\repositories\BO4E-python\tests\test_angebot.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\..\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests\test_angebot.py:12: in <module>
    from tests.test_adresse import example_adresse
tests\test_adresse.py:4: in <module>
    from py._path.local import LocalPath  # type:ignore[import]
E   ModuleNotFoundError: No module named 'py._path'; 'py' is not a package
_______________________________________________________________________________ ERROR collecting tests/test_ausschreibung.py ________________________________________________________________________________ 
ImportError while importing test module 'C:\Users\marce\repositories\BO4E-python\tests\test_ausschreibung.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\..\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests\test_ausschreibung.py:13: in <module>
    from tests.test_adresse import example_adresse
tests\test_adresse.py:4: in <module>
    from py._path.local import LocalPath  # type:ignore[import]
E   ModuleNotFoundError: No module named 'py._path'; 'py' is not a package
____________________________________________________________________________ ERROR collecting tests/test_ausschreibungsdetail.py ____________________________________________________________________________ 
ImportError while importing test module 'C:\Users\marce\repositories\BO4E-python\tests\test_ausschreibungsdetail.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\..\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests\test_ausschreibungsdetail.py:15: in <module>
    from tests.test_adresse import example_adresse
tests\test_adresse.py:4: in <module>
    from py._path.local import LocalPath  # type:ignore[import]
E   ModuleNotFoundError: No module named 'py._path'; 'py' is not a package
_____________________________________________________________________________ ERROR collecting tests/test_ausschreibungslos.py ______________________________________________________________________________ 
ImportError while importing test module 'C:\Users\marce\repositories\BO4E-python\tests\test_ausschreibungslos.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\..\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests\test_ausschreibungslos.py:12: in <module>
    from tests.test_ausschreibungsdetail import example_ausschreibungsdetail, example_ausschreibungsdetail_dict
tests\test_ausschreibungsdetail.py:15: in <module>
    from tests.test_adresse import example_adresse
tests\test_adresse.py:4: in <module>
    from py._path.local import LocalPath  # type:ignore[import]
E   ModuleNotFoundError: No module named 'py._path'; 'py' is not a package
_____________________________________________________________________________ ERROR collecting tests/test_geschaeftspartner.py ______________________________________________________________________________ 
ImportError while importing test module 'C:\Users\marce\repositories\BO4E-python\tests\test_geschaeftspartner.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\..\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests\test_geschaeftspartner.py:4: in <module>
    from py._path.local import LocalPath  # type:ignore[import]
E   ModuleNotFoundError: No module named 'py._path'; 'py' is not a package
========================================================================================== short test summary info ========================================================================================== 
ERROR tests/test_adresse.py
ERROR tests/test_angebot.py
ERROR tests/test_ausschreibung.py
ERROR tests/test_ausschreibungsdetail.py
ERROR tests/test_ausschreibungslos.py
ERROR tests/test_geschaeftspartner.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 6 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 
============================================================================================= 6 errors in 1.15s ============================================================================================= 
```

## Additional context

Please state your **Python version**:

- 3.10.10
- 3.11.3 (equivalent errors)

### Tested fixes

1. Manually adding the py module to the tests env dependencies fixes the issue.
2. Replacing the LocalPath with Path

As the ``py``-module is [advising](https://py.readthedocs.io/en/latest/path.html) to switch to the standard lib and ``LocalPath`` is only used in two lines, we should choose the latter.
> Note: The ‘py’ library is in “maintenance mode” and so is not recommended for new projects. Please check out pathlib or pathlib2 for path operations.

**Comment by hf-kklein** [comment]
Seems like an auto completion gone wrong in the IDE.

**Comment by hf-kklein** [comment]
fixed in #468
